scripts/heroku-build

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# Build a Heroku app slug using the Heroku (classic) buildpack interface.
+# <https://devcenter.heroku.com/articles/buildpack-api>
+#
+set -euo pipefail
+
+cd "$(dirname "$0")/.."
+
+echo "---> Hello from $0"
+echo "     Source directory is $PWD"
+echo "     Build  directory is $PWD/build"
+echo
+echo "---> Removing any previous build/app/"
+rm -rf build/app/
+
+echo
+echo "---> Ensuring build dirs exist"
+mkdir -vp build/{app,cache,env}
+
+echo
+echo "---> Copying source to build/app/"
+# Don't use `git archive` because it excludes uncommitted (but unignored) files
+# in the working tree.  Pass thru tar so that it handles directory creation for
+# us.
+git ls-files --cached --others --exclude-standard -z \
+  | tar --create  --file - --null --files-from - \
+  | tar --extract --file - --verbose -C build/app/
+
+echo
+echo "---> Setting buildpack env vars from source"
+export SOURCE_VERSION SOURCE_DESCRIPTION
+echo "     SOURCE_VERSION=${SOURCE_VERSION:=$(git describe --always --dirty)}"
+echo "     SOURCE_DESCRIPTION=${SOURCE_DESCRIPTION:=$(git log -1 --format=%s)}"
+
+if [[ ! -d build/pack ]]; then
+  echo
+  echo "---> Cloning buildpack into build/pack/"
+  git clone --depth 1 https://github.com/heroku/heroku-buildpack-nodejs build/pack
+else
+  echo
+  echo "---> Using buildpack in build/pack/"
+fi
+
+echo
+echo "---> Buildpack version is $(git -C build/pack describe --always --dirty)"
+
+echo
+echo "---> Running buildpack"
+docker run \
+  --rm --interactive $(tty --quiet && echo --tty) \
```

Comment:
When running locally, I see an error for the ancient macOS tty
```
tty: illegal option -- -
usage: tty [-s]
```

Updating to `tty -s` works as expected

Reply:
Ah, I'll fix this up. My preference is use to long options in code so that they're more self-describing, but that runs up against OS/version compatibility where short options are often more widely supported.

Reply:
https://github.com/nextstrain/nextstrain.org/pull/859